### PR TITLE
Additional CSS panel in Customizer: change link from Codex to HelpHub

### PR DESCRIPTION
**Pull request by stevenlinx**
In the "Additional CSS panel" in Customizer, the linked text "Learn more about CSS" is linking to the Codex page.
https://codex.wordpress.org/CSS

There is now an equivalent page on HelpHub so the link shall be updated.
https://wordpress.org/support/article/css/

Additional CSS panel in Customizer

Trac ticket: https://core.trac.wordpress.org/ticket/55710

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

File: src/wp-includes/class-wp-customize-manager.php [hunk 1]
```diff
@@ -5673,7 +5673,7 @@ public function register_controls() {
 		$section_description .= __( 'Add your own CSS code here to customize the appearance and layout of your site.' );
 		$section_description .= sprintf(
 			' <a href="%1$s" class="external-link" target="_blank">%2$s<span class="screen-reader-text"> %3$s</span></a>',
```
Comment (audrasjb) [inline src/wp-includes/class-wp-customize-manager.php:5675]: Ideally I'd add a translator comment for the 3 placeholder :)

**Review by audrasjb — CHANGES_REQUESTED** [review]
Added a suggestion for some translator comments (document placeholders)

**Comment by stevenlinx** [comment]
Updated with a translator comment.

File: src/wp-includes/class-wp-customize-manager.php (hunk 1 above)
Comment (SergeyBiryukov) [reply src/wp-includes/class-wp-customize-manager.php:5675]: The line with the placeholders here is not translatable, so it does not need a translator comment. The first commit in this PR actually looks good to me as is :)

File: src/wp-includes/class-wp-customize-manager.php [hunk 2]
```diff
@@ -5672,10 +5672,10 @@ public function register_controls() {
 		$section_description  = '<p>';
 		$section_description .= __( 'Add your own CSS code here to customize the appearance and layout of your site.' );
 		$section_description .= sprintf(
+			/* translators: 1: HelpHub CSS article URL 2: Title description of the article 3: Accessibility text. */
 			' <a href="%1$s" class="external-link" target="_blank">%2$s<span class="screen-reader-text"> %3$s</span></a>',
-			esc_url( __( 'https://codex.wordpress.org/CSS' ) ),
+			esc_url( __( 'https://wordpress.org/support/article/css/' ) ),
 			__( 'Learn more about CSS' ),
-			/* translators: Accessibility text. */
```
Comment (SergeyBiryukov) [inline src/wp-includes/class-wp-customize-manager.php:5678]: The "Accessibility text" translator comment should not be removed.

**Review by SergeyBiryukov — APPROVED** [review]

**Comment by SergeyBiryukov** [comment]
Thanks for the PR! Merged in https://core.trac.wordpress.org/changeset/53393.

**Pull request closed without merging**